app/assets/stylesheets/links.scss

```diff
@@ -27,6 +27,18 @@ a.nav-link.active {
   outline: none;
 }
 
```

Comment:
There are some links like the change energy usage units which needs fixing - plus work out what to do with sub nav